docs/content/posts/2024-03-xx-release-1.19.0.md

```diff
@@ -0,0 +1,466 @@
+</div>
+
+**More Information**
+* [minibatch-regular-joins](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/dev/table/tuning/#minibatch-regular-joins).
+* [FLIP-415: Introduce a new join operator to support minibatch](https://cwiki.apache.org/confluence/display/FLINK/FLIP-415%3A+Introduce+a+new+join+operator+to+support+minibatch)
+
+# Key Improvements
+
+## Runtime & Coordination
+
+### Support dynamic source parallelism inference for batch jobs
+
+In Flink 1.19, we have supported dynamic source parallelism inference for batch jobs, which allows
+source connectors to dynamically infer the parallelism based on the actual amount of data to consume.
+This feature is a significant improvement over previous versions, which only assigned a fixed default
+parallelism to source vertices.
+Source connectors need to implement the inference interface to enable dynamic parallelism inference.
+Currently, the FileSource connector has already been developed with this functionality in place.
+Additionally, the configuration `execution.batch.adaptive.auto-parallelism.default-source-parallelism`
+will be used as the upper bound of source parallelism inference. And now it will not default to 1.
+Instead, if it is not set, the upper bound of allowed parallelism set via
+`execution.batch.adaptive.auto-parallelism.max-parallelism` will be used. If that configuration is
+also not set, the default parallelism set via `parallelism.default` or `StreamExecutionEnvironment#setParallelism()`
+will be used instead.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/deployment/elastic_scaling/#enable-dynamic-parallelism-inference-support-for-sources).
+* [FLIP-379: Support dynamic source parallelism inference for batch jobs](https://cwiki.apache.org/confluence/display/FLINK/FLIP-379%3A+Dynamic+source+parallelism+inference+for+batch+jobs)
+
+### Support standard YAML for FLINK configuration
+
+Starting with Flink 1.19, Flink has officially introduced full support for the standard YAML 1.2
+syntax. The default configuration file has been changed to `config.yaml` and placed in the `conf/`
+directory. Users should directly modify this file to configure Flink.
+If users want to use the legacy configuration file `flink-conf.yaml`, users just need to copy this
+file into the `conf/` directory. Once the legacy configuration file `flink-conf.yaml` is detected,
+Flink will prioritize using it as the configuration file.And in the upcoming Flink 2.0, the
+`flink-conf.yaml` configuration file will no longer work.
+
+**More Information**
+* [flink-configuration-file](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/#flink-configuration-file)
+* [FLIP-366: Support standard YAML for FLINK configuration](https://cwiki.apache.org/confluence/display/FLINK/FLIP-366%3A+Support+standard+YAML+for+FLINK+configuration?src=contextnavpagetreemode)
+
+### Profiling JobManager/TaskManager on Flink Web
```

Comment:
@yuchen-ecnu I've added two imgs based on a testing job via latest rc1 package, please help to review the content, thanks!
![image](https://github.com/apache/flink-web/assets/3712895/e1688d9b-fc1b-4544-9e58-ef096dea426b)